config/manager/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ resources:
 images:
 - name: controller
   newName: docker.io/projectcontour/contour-operator
-  newTag: main
+  newTag: bbfcec2c
```

Comment:
guessing this should be reverted :)

Reply:
@skriss thanks for catching, I reverted in commit `a154c90`.